--- plasma-remotecontrollers-5.27.5/CMakeLists.txt~	2023-05-09 11:38:31.000000000 +0000
+++ plasma-remotecontrollers-5.27.5/CMakeLists.txt	2023-05-13 17:31:58.848889239 +0000
@@ -67,7 +67,7 @@
 
 install(FILES plasma-remotecontrollersrc DESTINATION ${KDE_INSTALL_CONFDIR})
 install(FILES plasma-remotecontrollers.notifyrc DESTINATION ${KDE_INSTALL_KNOTIFYRCDIR})
-install(FILES 40-uinput.rules DESTINATION ${KDE_INSTALL_LIBDIR}/udev/rules.d)
+install(FILES 40-uinput.rules DESTINATION /lib/udev/rules.d)
 
 install(FILES org.kde.plasma.remotecontrollers.metainfo.xml DESTINATION ${KDE_INSTALL_METAINFODIR})
 
